test/spec.yml

```diff
@@ -1373,3 +1374,346 @@
     -  in: bn create node8 testArrayOpts --config=aaa --config=bbb
        exit: 1
        err: "Non object create configuration options: aaa,bbb"
+
+- test: Test new trigger (good-path)
+  cleanup:
+    - bn trigger remove testtrigger
+    - bn remove testtriggerfunc
+  steps:
+    - in: bn create node8 testtriggerfunc
+      out: |-
```

Comment:
Not worth testing, it's tested in the functions tests.

Reply:
removed